plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -172,15 +172,31 @@ func (l *persistentVolumeLabel) findVolumeLabels(volume *api.PersistentVolume) (
 	existingLabels := volume.Labels
 
 	// All cloud providers set only these two labels.
-	domain, domainOK := existingLabels[v1.LabelFailureDomainBetaZone]
-	region, regionOK := existingLabels[v1.LabelFailureDomainBetaRegion]
+	topologyLabelGA := true
+	domain, domainOK := existingLabels[v1.LabelTopologyZone]
+	region, regionOK := existingLabels[v1.LabelTopologyRegion]
+	// If they dont have GA labels we should check for failuredomain beta labels
+	// TODO: remove this once all the cloud provider change to GA topology labels
```

Comment:
> remove this once all the cloud provider change to GA topology labels

When will cloud provider change to GA topology labels, for in-tree driver, it's already in maintenance mode, so it would always use beta label? While for CSI driver, it won't use these labels.

Reply:
We will move in-tree to GA label in this quarter as CSI migration is likely slip and the Beta label will probably be removed this or next release. I will make a PR about making gce-pd label to GA after this PR merged.